jsonObject.class.asp

```diff
@@ -1114,6 +1117,62 @@ class JSONarray
 end class
 
 
+class JSONscript
+	dim i_parent, i_depth, i_version
+	dim s_value, s_nullString
+
+	' The value
+	public property get value
+		value = s_value
+	end property
+	
+	public property let value(newValue)
+		if (len(newValue) = 0) then newValue = s_nullString

```

Comment:
A type check would be better to ensure that only strings and `Null` are valid before checking the length.